saskatoon/member/admin.py

```diff
@@ -259,13 +270,35 @@ class PersonAdmin(admin.ModelAdmin):
         'auth_user__email',
     )
 
+    @admin.display(description="AuthUser")
+    def authuser(self, person):
+        try:
+            user = person.auth_user
+            url = reverse('admin:member_authuser_change', kwargs={'object_id': user.id})
+            return mark_safe(f"<a href={url}>{user.email}</a>")
+        except AuthUser.DoesNotExist:
+            return None
+
     def get_queryset(self, request):
         queryset = super().get_queryset(request)
         queryset = queryset.annotate(
             postal_code_cleaned=Replace('postal_code', Value(" "), Value(""))
         )
         return queryset
 
+    @admin.action(description="Create missing user(s)")
+    def create_user(self, request, queryset):
+        """Associate Person with new AuthUser object"""
+        nb_users = 0
+        for person in queryset.filter(auth_user__isnull=True):
+            email = f"FAKE@person.{person.pk}"
+            AuthUser.objects.create(email=email, person=person)
+            nb_users += 1
+        messages.add_message(request, messages.SUCCESS,
```

Comment:
This looks useful, but I wonder if we're not end up with many FAKE authuser in the DB. It's probably better not have them at all I supose.

Reply:
The problem when a Person is not associated with a AuthUser is that their email cannot be set, which is what core members seemed to complain about (i.e. owner's email only showing in the comments). We could look into a special button to create this AuthUser on demand (only when a person without an email needs its' email field to be updated) but honestly I think we should be consistent throughout the database and each Person should be associated with an AuthUser. Only pickleader and core members will have password sets though.

Reply:
I should also probably find a way to look for `pending_contact_email` values that would be hidden in some Property models (pending fields only show when property is pending) I'm sure there are some owner's email addresses that were not properly entered because the process to create a Person/AuthUser pair was not clear (looks like the comment section was used for this most of the time).